src/app/services/age-verification/age-verification.service.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { TestBed } from '@angular/core/testing';
+
+import { AgeVerificationService } from './age-verification.service';
+
+describe('AgeVerificationService', () => {
+  let service: AgeVerificationService;
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+
+    service = TestBed.get(AgeVerificationService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should contains "is legal age" when user is legal age', () => {
+    const params = {
```

Comment:
create a base params called `userWithoutAge` and add here inside each test:

```typescript
beforeEach(() => {
userWithoutAge = {
  firstName: 'Lucas',
  lastName: 'Monstro'
};

...
});
```

Inside each test:
```
const user = { ...userWithoutAge, age: 7 };
const user = { ...userWithoutAge, age: 19 };
```